EMS/common-bundle/src/Common/CoreApi/Endpoint/Search/Search.php

```diff
@@ -118,4 +118,40 @@ public function getDocument(string $index, ?string $contentType, string $id, arr
             'sources-excludes' => $sourcesExcludes,
         ])->getData());
     }
+
+    /**
+     * @param string[] $aliases
+     *
+     * @return array<string, array<int, string>>
+     */
+    public function getIndicesForContentTypes(array $aliases): array
```

Comment:
Method should be added to: EMS\CommonBundle\Contracts\CoreApi\Endpoint\Search\SearchInterface

Reply:
713326cd1c29d9d19f63abc95baba545975667b2